### PR TITLE
Add logo to docs

### PR DESCRIPTION
**Pull request by clizbe**
Part of spine-tools/SpineInterface.jl#112

image

## Checklist before merging
- [X] Documentation is up-to-date
- [NA] Unit tests have been added/updated accordingly
- [NA] Code has been formatted according to SpineOpt's style
- [NA] Unit tests pass

### CODE REVIEW

**Comment by clizbe** [comment]
@abelsiqueira I just noticed my formatter seems to have edited a bunch of docs/make.jl.
The SpineOpt way is to have spaces on either side of operators - how do I tell my formatter to do something different between SpineOpt and Tulipa?

**Comment by clizbe** [comment]
I don't know what you mean by "having the project open."
And I didn't explicitly run JuliaFormatter - it must have run itself on Save.
But I was hoping those settings were isolated to Tulipa... because now my Tulipa formatter is probably running rampant on my SpineOpt code. :/

**Comment by clizbe** [comment]
Sorry if that sounded angry. Not angry, just tired and confused. XD

**Comment by abelsiqueira** [comment]
PS. You can disable "Save on format" just for SpineOpt.jl: Open the settings and click on "Workspace", search for format on save and disable it.

**Comment by clizbe** [comment]
Yeah I asked the guys and they said they always disable "Run on Save" when they work on SpineOpt. :(
Something we can change once #517 is hopefully implemented.

File: docs/Project.toml [hunk 1]
```diff
@@ -2,4 +2,5 @@
 CSV = "336ed68f-0bac-5ca0-87d4-7b16caf5d00b"
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
```
Comment (manuelma) [inline docs/Project.toml:5]: Do we need Revise as a dependency for our project?

Comment (clizbe) [reply docs/Project.toml:5]: Ooh nice catch! No we don't.

Comment (clizbe) [reply docs/Project.toml:5]: Fixed it: #900

**Pull request merged**